tomcat/tomcat-jta/src/main/java/org/jboss/narayana/tomcat/jta/NarayanaJtaServletContextListener.java

```diff
@@ -64,6 +66,23 @@
     private static final List<String> DEFAULT_EXPIRY_SCANNERS = Arrays
             .asList(ExpiredTransactionStatusManagerScanner.class.getName());
 
+    /**
+     * Initialize and start Narayana JTA services.
+     * <p>
+     * During the setup node identifier, recovery modules, orphan filters, and expiry scanners are setup. Configuration file
```

Comment:
@ochaloup Ah I was waiting for you to respond to this question but I see you added another commit that includes the link. In future if you ack/nack each comment or include a catch all comment then that will let the reviewer know that the PR is ready to be reviewed again.